README.md

```diff
@@ -1,57 +1,29 @@
-# Project Name
+# Apple.com Webpage Clone
 
-> One paragraph statement about the project.
+> This project was to clone the Apple.com webpage using backgrounds and gradient colors.
 
-![screenshot](./app_screenshot.png)
+![screenshot](/images/apple-webpage.png)
 
-Additional description about the project and its features.
+This project contains the following features:
+
+1.  Navbar with gradient apple.com colors
+2.  Grid and FlexBox layouts
+3.  Background image
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML5 and CSS
+- Font Awesome Icons
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+[Live Demo Link](https://azeem838.github.io/apple.com-webpage/)
```

Comment:
# [REQUIRED] - FIX - Live link to your repo.
The link is not working. Kindly fix that,